### PR TITLE
[CLOUDTRUST-2296] Add endpoint execute-actions-email in account client

### PR DESCRIPTION
**Pull request by fperot74**
Use new method implemented in cloudtrust/keycloak-rest-api-extensions#26

(+ more than 60 "lints" removed)

### CODE REVIEW

File: account.go [hunk 1]
```diff
@@ -7,10 +7,10 @@ import (
 )
 
 const (
-	accountPath                        = "/auth/realms/:realm/account"
 	accountExtensionAPIPath            = "/auth/realms/master/api/account/realms/:realm"
-	accountPasswordPath                = accountExtensionAPIPath + "/credentials/password"
```
Comment (bsoniam) [inline account.go]: why was this removed?

Comment (fperot74) [reply account.go]: No just moved 3 lines below... It was not logical to do accountExtensionAPIPath + "/credentials... rather than accountCredentialsPath + "...

Comment (fperot74) [reply account.go]: Not removed, just moved 3 lines below... It was not logical to do accountExtensionAPIPath + "/credentials... rather than accountCredentialsPath + "...

Comment (bsoniam) [reply account.go]: Sorry, I did not see this
